remote_llm.py

```diff
@@ -32,15 +33,27 @@ def generate_text(preprompt, prompt, model):
     if token is None:
         raise ValueError('LLM Verification Router token is not set')
     
-    log.info(f'Received text generation request for prompt "{prompt}" for model {model}')
+    log.info(f'Received text generation request for prompt "{prompt}" for model {model} at url {url}')
     # Load the Vanilla Neox API key from the config file.
 
     raw_response = post(url=route,
                         headers={'Authorization': f'Bearer {token}'},
-                        json={'prompt': prompt, "preprompt" : preprompt, "model": model})
+                        
+                        # UUID needs to be provided here
+                        # https://github.com/aivillage/llm_router/blob/main/src/chat/mod.rs#L35
+                        json={"uuid": 1234, 'prompt': prompt, "preprompt" : preprompt, "model": model})
```

Comment:
This was the issue UUID was not provided so router would throw an exception and not return a json

![image](https://github.com/aivillage/llm_verification/assets/7213793/fe09d602-15d0-496a-835c-f7a0501be61b)

Reply:
Opening a new comment